lib/src/includes/signed_video_sign.h

```diff
@@ -336,16 +336,32 @@ signed_video_set_end_of_stream(signed_video_t *self);
  *       != SV_OK) {
  *     // Handle error
  *   }
- *
+ *   // Apply any configuration of choice.
  *   status = signed_video_generate_golden_sei(sv);
  *   if (status != SV_OK) {
  *     // Handle error
  *   }
+ *   // All necessary configurations need to be done prior to this call.
```

Comment:
You do not have to be this explicit. In fact, if the user really wants true code he/she can look in the test files for example. It is open-source. It is enough to say that "The user can get the golden SEI using signed_video_get_sei()."

Also, you should mention that the golden SEI in general is not generated instantly, hence the user has to wait for it. You can in the example code put a sleep(1) or something.